docs/UserGuide.md

```diff
@@ -124,6 +124,10 @@ The `Tool Bar` at the top provides a set of configuration options that control t
     * `Day`: commits within a day (commits made within 00:00 to 23:59) are shown as one ramp
     * `Week`: commits within a week are shown as one ramp
 * `Since`, `Until` : the date range for the Ramp Chart (not applied to the Contribution Bars).
+* `Reset date range` : this will reset the date range of the Ramp Chart to the default date range.
+* `Breakdown by file format` : this will toggle the contribution bar to either display the bar by 
+    * the total lines of codes added (if checkbox is left unchecked), or
+    * a breakdown of the number of lines of codes added to each file format (if checkbox is checked). 
```

Comment:
Might be better if a similar format to `Search` is used, for `Reset date range` and `Breakdown by file format`

e.g.
> ~~this will~~ reset**s** the date range of the Ramp Chart